contracts/contracts/buyback/BaseBuyback.sol

```diff
@@ -52,8 +52,11 @@ abstract contract BaseBuyback is Initializable, Strategizable {
     address public immutable cvx;
     address public immutable cvxLocker;
 
-    // Ref: https://docs.uniswap.org/contracts/universal-router/technical-reference#command-structure
-    bytes private constant swapCommand = hex"0000";
+    // Amount of `oTokens` to use for OGV buyback
+    uint256 public ogvShare;
+
+    // Amount of `oTokens` to use for CVX buyback
+    uint256 public cvxShare;
```

Comment:
`ogvShare` and ` cvxShare` could be packed together using two. uint128.

Reply:
This makes sense. But, right now, this contract is used only once every two weeks. I'd prefer keeping this a bit simple rather than cast into `uint128` and making the code a bit more messy for gas usage

Reply:
understandable.